helpers/database-import-scripts/rnacentral/rfam_model_lengths_15.0.txt

```diff
@@ -0,0 +1,4178 @@
+RF00001	120
```

Comment:
that file should go to DBs repo I think and be imported as pipeline input

Reply:
Yeah, it could be moved, I’ll need to think about it. Right now, it’s used in a post-processing step (RNAcentral), not in the main pipeline. I originally added it to the repo so users could access it easily, rather than having to download it from FTP or regenerate it themselves since it's small and used in a separate step. I’ll revisit this for the next release when I move the configs to a separate repo.